### PR TITLE
Create option for admin to undo final decision

### PR DESCRIPTION
**Pull request by Jeff-Thompson12**
The code is meant to address #146. It seeks to accomplish this feat in the following ways. 

1. Replaces the "Submit Decision" button for admin users with an "Undo Decision" button when a final decision has been made for a package.
2. Sets the decision value for the package to '', which is equivalent to missing/NULL.
3. Sends a notification that a final decision was undone to the logging function.

The code utilizes the formatting and structure already in place for submitting a decision instead of creating new classes.

### CODE REVIEW

**Review by Robert-Krajcik — APPROVED** [review]
I logged in as a user and set the overall comment and overall risk for the `dplyr` package.
image

Then I logged in as an admin and checked the dplyr package
image

I see the "submit decision" was replaced with an "undo decision" button.
Only an admin can see it. 
Then I clicked on "undo decision".  Nice modal.
image

It works!

**Comment by marlycormar** [comment]
The PR looks good to me! 

There is a major refactoring happening on #162, so it'll be a good idea to wait for that one to be merged before merging this R. PR #162 will force some changes on this PR. @Robert-Krajcik do you agree?

**Comment by Robert-Krajcik** [comment]
Sounds good to me, @marlycormar 
Let's hold this until #162 has been merged then.

**Comment by Jeff-Thompson12** [comment]
Closing this PR because of changes in the dev branch. Will open a new one with updated code.

**Pull request closed without merging**
